### DISCUSSION TITLE
Noob question. How can I access the `editor` of jodit on startup? :)

### DISCUSSION THREAD

**Discussion by ptatoChi** [julianpoemp/ngx-jodit#20] [Q&A] [answered]
Hi! I migrated from jodit-angular. I used this library and its been great so far. Now it seems i need to use the editor for some things. how can i access that?

**Comment by julianpoemp** [comment, marked as answer]
> Hi! I migrated from jodit-angular. I used this library and its been great so far. Now it seems i need to use the editor for some things. how can i access that?

Hi,
ngx-jodit offers an attribute called "jodit". Just assign a ViewChild to your ngx-jodit component. E.g.:

````html
<ngx-jodit #ngxJodit></ngx-jodit>
````


````typescript

@ViewChild("ngxJodit") ngxJodit: NgxJoditComponent;

...

ngAfterViewInit(){
this.ngxJodit.jodit // <- Jodit instance with all features.
}

````

**Comment by ptatoChi** [comment]
Thank you!!
